### PR TITLE
mgr/dashboard: 'destroyed' view in CRUSH map viewer

### PR DESCRIPTION
**Pull request by avanthakkar**
Fixes: https://tracker.ceph.com/issues/44198

Signed-off-by: Avan Thakkar <athakkar@redhat.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by Devp00l** [comment]
jenkins test make check

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
                   (nodeSelected)="onNodeSelected($event)">
               <ng-template let-node>
                 <span class="badge"
-                      [ngClass]="{'badge-success': ['in', 'up'].includes(node.status), 'badge-danger': ['down', 'out'].includes(node.status)}">
+                      [ngClass]="{'badge-success': ['in', 'up'].includes(node.status), 'badge-danger': ['down', 'out','destroyed'].includes(node.status)}">
```
Comment (votdev) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.html]:
```suggestion
                      [ngClass]="{'badge-success': ['in', 'up'].includes(node.status), 'badge-danger': ['down', 'out', 'destroyed'].includes(node.status)}">
```

**Comment by votdev** [comment]
I think you need to sign the second commit or squash the PR to make the Sign-off-by test happy.

**Comment by avanthakkar** [comment]
> I think you need to sign the second commit or squash the PR to make the Sign-off-by test happy.

Yes, will do that

**Comment by votdev** [comment]
jenkins test dashboard

**Pull request merged**